pegasus/sites.v3/hourofcode.com/views/theme_common_head_after.haml

```diff
@@ -11,3 +11,18 @@
     });
     $('#language-dropdown').attr('style', 'visibility: visible;');
   });
+
+  $(document).ready(function() {
+    // When nav hamburger is clicked, toggle responsive menu
+    $("#hoc-nav-hamburger").click(function(e){
+        $('#hoc-nav ul').toggleClass('hide-responsive-menu');
+        e.preventDefault();
+    });
+
+    // When clicking anywhere outside the responsive menu, dismiss the menu
+    $(document).click(function(event) {
+      if(!$(event.target).closest('#hoc-nav-hamburger').length) {
+        $('#hoc-nav ul').addClass('hide-responsive-menu');
```

Comment:
Interesting technique.. have we used it elsewhere?  does it work okay for touch devices?